internal/pkg/cli/env_init.go

```diff
@@ -666,14 +678,15 @@ func (o *initEnvOpts) deployEnv(app *config.Application,
 			DNSName:             app.Domain,
 			AccountPrincipalARN: caller.RootUserARN,
 		},
-		Prod:                 o.isProduction,
-		AdditionalTags:       app.Tags,
-		CustomResourcesURLs:  customResourcesURLs,
-		ArtifactBucketARN:    artifactBucketARN,
+		Prod:                o.isProduction,
+		AdditionalTags:      app.Tags,
+		CustomResourcesURLs: customResourcesURLs,
+    ArtifactBucketARN:    artifactBucketARN,
```

Comment:
nit: we can keep this line aligned with the rest 👀

Reply:
ooh I was so confused how this happened but it's because of the commit that merged mainline to this branch.

Fixed!